pom.xml

```diff
@@ -265,6 +265,7 @@
         <version.com.google.guava>25.0-jre</version.com.google.guava>
         <version.com.h2database>1.4.197</version.com.h2database>
         <version.com.microsoft.azure>6.1.0</version.com.microsoft.azure>
+        <version.com.nimbus.jose-jwt>5.4</version.com.nimbus.jose-jwt>
```

Comment:
This is a couple years old; project is up to 8.11

https://mvnrepository.com/artifact/com.nimbusds/nimbus-jose-jwt

If this is the one we need, ok, but I'm checking as we'll get prompts to upgrade just because, and if security scanners identify anything with this one we'll need to deal with it.

Reply:
I will quickly fix that one, once we move to a later version of SmallRye JWT we can likely drop it as it is just being used for the token generation on the client side of the test.